buildtools/mimalloc/Dockerfile

```diff
@@ -9,6 +9,6 @@ RUN mkdir -p /mimalloc && curl -L https://github.com/microsoft/mimalloc/archive/
 WORKDIR /mimalloc
 ADD mimalloc.patch mimalloc.patch
 RUN patch -p1 < mimalloc.patch
-RUN mkdir -p out/release && cd out/release && cmake ../.. -DMI_BUILD_SHARED=off -DMI_BUILD_TESTS=off && make && llvm-ranlib-14 libmimalloc.a
+RUN mkdir -p out/release && cd out/release && cmake ../.. -DMI_BUILD_SHARED=off -DMI_BUILD_TESTS=off && make && llvm-ranlib-15 libmimalloc.a
```

Comment:
I guess we should link the llvm version with https://github.com/corazawaf/coraza-proxy-wasm/pull/54/files#diff-83d9e656ee6391e146dd2a479f814494e313b923cfc227c2a630bd1fb9f9c5dbR9

Reply:
Found that cmake ignores the environment variable we set, but we can use it here to remove that version dependency

https://gitlab.kitware.com/cmake/cmake/-/issues/18277